src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -114,6 +114,11 @@ public bool HasAssemblies
             get { return Model.LoadedAssemblies.Any(); }
         }
 
+        public bool CanPublish
```

Comment:
maybe you can replace the `CanPublishNewPackage` and `CanPublishNewPackageVerison` methods with this new one ?

Reply:
Yes, good idea.